app/routing/VersionRoutingRequestHandler.scala

```diff
@@ -43,30 +42,45 @@ class VersionRoutingRequestHandler @Inject() (versionRoutingMap: VersionRoutingM
       filters = filters.filters
     ) {
 
-  private val featureSwitch = FeatureSwitches(config.featureSwitches)
-
   private val unsupportedVersionAction = action(Results.NotFound(Json.toJson(UnsupportedVersionError)))
 
+  private val resourceNotFoundAction = action(Results.NotFound(NotFoundError.asJson))
+
   private val invalidAcceptHeaderError = action(Results.NotAcceptable(Json.toJson(InvalidAcceptHeaderError)))
```

Comment:
Need to update this to `private val invalidAcceptHeaderError = action(Results.NotAcceptable(InvalidAcceptHeaderError.asJson))`